tasklist/client/src/Layout/Header/styles.module.scss

```diff
@@ -15,3 +15,7 @@
     color: var(--cds-link-primary);
   }
 }
+
+.languageDropdownPadding {
```

Comment:
@Skaiir [Please use the spacing tokens provided by Carbon](https://carbondesignsystem.com/elements/spacing/overview/)

Also remove the quotes since that's not valid CSS